spring-integration-cassandra/src/main/resources/org/springframework/integration/config/xml/spring-integration-cassandra-1.0.xsd

```diff
@@ -17,5 +17,187 @@
+				Message into Cassandra cluster
+			</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:extension base="outboundType">
+					<xsd:choice minOccurs="0" maxOccurs="2">
+					<xsd:element name="parameter-expression" type="queryParameterType" minOccurs="0" maxOccurs="unbounded">
+					<xsd:annotation>
+						<xsd:documentation>
+							Specify an expression for parameter variable placeholder in cql statement.
+						</xsd:documentation>
+					</xsd:annotation>
+				    </xsd:element>
+					<xsd:element ref="integration:poller" minOccurs="0" maxOccurs="1"/>
+					<xsd:element name="request-handler-advice-chain" type="integration:handlerAdviceChainType" minOccurs="0" maxOccurs="1" />
+					</xsd:choice>
+				</xsd:extension>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	
+	<xsd:element name="outbound-gateway">
+		<xsd:annotation>
+			<xsd:documentation>
+				Defines cassandra outbound gateway that
+				writes the contents of the
+				Message into Cassandra cluster
+			</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:extension base="outboundType">
+					<xsd:choice minOccurs="0" maxOccurs="2">
+					<xsd:element name="parameter-expression" type="queryParameterType" minOccurs="0" maxOccurs="unbounded">
+					<xsd:annotation>
+						<xsd:documentation>
+							Specify an expression for parameter variable placeholder in cql statement.
+						</xsd:documentation>
+					</xsd:annotation>
+				    </xsd:element>
+					<xsd:element name="request-handler-advice-chain" type="integration:handlerAdviceChainType" minOccurs="0" maxOccurs="1" />
+					</xsd:choice>
+					<xsd:attribute name="reply-channel" type="xsd:string">
+						<xsd:annotation>
+							<xsd:documentation>
+								Message Channel to which replies should be sent after being received from Cassandra cluster.
+							</xsd:documentation>
+							<xsd:appinfo>
+								<tool:annotation kind="ref">
+									<tool:expected-type
+										type="org.springframework.messaging.MessageChannel"/>
+								</tool:annotation>
+							</xsd:appinfo>
+						</xsd:annotation>
+					</xsd:attribute>
+					<xsd:attribute name="input-channel" type="xsd:string">
+						<xsd:annotation>
+							<xsd:documentation>
+								Message Channel to which Messages should be sent to Cassandra.
+							</xsd:documentation>
+							<xsd:appinfo>
+								<tool:annotation kind="ref">
+									<tool:expected-type
+										type="org.springframework.messaging.MessageChannel"/>
+								</tool:annotation>
+							</xsd:appinfo>
+						</xsd:annotation>
+					</xsd:attribute>
+				</xsd:extension>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	
+	<xsd:complexType name="outboundType">
+		<xsd:annotation>
+			<xsd:documentation>
+				Common configuration for cassandra adapters.
+			</xsd:documentation>
+		</xsd:annotation>
+		<xsd:attributeGroup ref="integration:channelAdapterAttributes" />
```

Comment:
Yes, we have to use this common group, but it is exactly for the `channel-adapter` definition. 
Should be moved there.
From other side we should mot forget about `lifecycle` and add `<xsd:attributeGroup ref="integration:smartLifeCycleAttributeGroup"/>` to the gateway anyway.